frontend/src/components/SelectAsync/index.jsx

```diff
@@ -34,18 +40,30 @@ export default function SelectAsync({
     }
   }, [value]);
 
+  const handleSelectChange = (newValue) => {
+    if (newValue === 'redirectURL') {
+      // Navigate to another page when "Add payment" is selected
+      history.push(urlToRedirect);
+    } else {
+      // Handle other select options
+      if (onChange) {
+        onChange(newValue[outputValue] || newValue);
+      }
+    }
+  };
+
   return (
     <Select
       loading={isLoading}
       disabled={isLoading}
       value={currentValue}
-      onChange={(newValue) => {
-        // setCurrentValue(newValue[outputValue] || newValue);
-        if (onChange) {
-          onChange(newValue[outputValue] || newValue);
-        }
-      }}
+      onChange={handleSelectChange}
     >
+      {selectOptions.length === 0 && withRedirect && (
+        <Select.Option key="redirectURL" value="redirectURL">
+          {labelText}
```

Comment:
could you add inside select button with icon (PlusCircleOutlined ) + labelText .

check antd 4.X docs

Reply:
![image](https://github.com/idurar/erp-crm/assets/100110395/37e1bd7d-32ec-4f0a-8a46-55b063c4f547)

added